.github/ISSUE_TEMPLATE.md

```diff
@@ -1,20 +1,70 @@
-**Please provide the following information**
+<!--
+###############################################################################
+  WARNING!
+  IGNORING THE FOLLOWING TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED
+  AS INCOMPLETE/INVALID
+###############################################################################
+-->
+
+## Checklist
+
+<!--
+################################## IMPORTANT ##################################
+
+As you read and fulfill each of the following requirements below,
+put an "x" (without the quotes) between the square brackets.
+
+When doing so, ensure that no space is left between the square brackets and the "x", like so: [x]
+
+Be honest!
+-->
+
+- [ ] I have read the **issue reporting guidelines** in the [contributing guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md), so I know how to submit a good report with the required information
+- [ ] I have verified that **the issue is not fixed and is reproducible** in the **[latest version](https://www.qbittorrent.org/download.php)**
+- [ ] I have **searched** the issue tracker for similar bug reports including closed ones **to avoid posting a duplicate issue report**
+- [ ] This is **not a support issue or question**, or the [wiki](https://github.com/qbittorrent/qBittorrent/wiki) did not contain the required information and the [forum](https://qbforums.shiki.hu/) and [subreddit](https://www.reddit.com/r/qBittorrent/) were not able to assist me, so I am asking here
+- [ ] I have **pasted/attached the settings file and relevant log(s)** (at least the most recent one) in the **Attachments** section at the bottom of the report and filled in all the other required information
+
+## Description
+
+<!--
+################################## IMPORTANT ##################################
+
+Delete each "(type here)" indicator and type your text in their place in the subsections below.
+
+Please make sure the description is worded well enough to be understood.
+Provide steps to reproduce the issue, any additional relevant information, suggested solution (if applicable) and as much context and examples as possible.
+For more information consult the Contributing Guidelines at https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md.
+
+Do not forget about the mandatory attachments!
+
+Use the Preview tab before posting to make sure your report looks like it is formatted properly.
+-->
+
+### qBittorrent version, install method (installer, compiled from git, ...), and operating system version
 
-### qBittorrent version and Operating System
 (type here)
 
-### If on linux, libtorrent-rasterbar and Qt version
+### If on Linux, `libtorrent-rasterbar` and `Qt` version
```

Comment:
@FranciscoPombal:  FYI, not everyone on Windows will be using the official builds.  Prior to the addition of official 64-bit builds, I was relying on the (now outdated) [<https://github.com/Gelmir/qBittorrent/releases>] repository for my Windows builds. 

So perhaps… 

> If on Linux **or using unofficial builds**, 

…would work better?

Reply:
I would remove "If on Linux" at all. I never remember what versions are used by official Windows/MacOS builds.

Reply:
There is an ongoing effort to make "official" nightly releases more easily available already: https://github.com/qbittorrent/qBittorrent/issues/12357. But it will still take a little while. When the time comes, we can make that adjustment in a separate PR. Thanks for the reminder though.